core/src/main/scala/org/apache/spark/scheduler/SparkListener.scala

```diff
@@ -227,6 +227,10 @@ case class SparkListenerUnschedulableTaskSetRemoved(
 @DeveloperApi
 case class SparkListenerBlockUpdated(blockUpdatedInfo: BlockUpdatedInfo) extends SparkListenerEvent
 
+@DeveloperApi
+case class MiscellaneousProcessAdded(time: Long, processId: String,
```

Comment:
I just realized we should name this SparkListenerMiscellaneousProcessAdded to differentiate from the cluster message, which we should call MiscellaneousProcessAdded

Reply:
Changed the name to SparkListenerMiscellaneousProcessAdded